### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rcaneill**
I copied from:
- https://github.com/pyOpenSci/software-peer-review/blob/main/appendices/review-template.md?plain=1
- https://github.com/openjournals/joss-reviews/blob/master/.buffy/templates/reviewer_checklist.md?plain=1

### CODE REVIEW

**Comment by rcaneill** [comment]
@vadmbertr @lesommer 
I propose that we do suggestions on this PR on which boxes we want to keep. I’ll reorder them afterward

File: README.md [hunk 1]
```diff
@@ -1,2 +1,93 @@
...
+## Checklist
+
+#### Documentation
+
+The package includes all the following forms of documentation:
+
+- [ ] **A statement of need** clearly stating problems the software is designed to solve and its target audience in README.
+- [ ] **Installation instructions:** for the development version of the package and any non-standard dependencies in README.
+- [ ] **Vignette(s)** demonstrating major functionality that runs successfully locally.
+- [ ] **Function Documentation:** for all user-facing functions.
```
Comment (rcaneill) [inline README.md]: these may be a little bit too much for us

File: README.md [hunk 2]
```diff
@@ -1,2 +1,93 @@
...
+
+The README should include, from top to bottom:
+
+- [ ] The package name
+- [ ] Badges for:
+    - [ ] Continuous integration and test coverage,
+    - [ ] Docs building (if you have a documentation website),
+    - [ ] A [repostatus.org](https://www.repostatus.org/) badge,
+    - [ ] Python versions supported,
+    - [ ] Current package version (on PyPI / Conda).
```
Comment (rcaneill) [inline README.md]: I would remove these

File: README.md [hunk 3]
```diff
@@ -1,2 +1,93 @@
...
+
+*NOTE: If the README has many more badges, you might want to consider using a table for badges: [see this example](https://github.com/ropensci/drake). Such a table should be more wide than high. (Note that the a badge for pyOpenSci peer-review will be provided upon acceptance.)*
+
+- [ ] Short description of package goals.
+- [ ] Package installation instructions
+- [ ] Any additional setup required to use the package (authentication tokens, etc.)
```
Comment (rcaneill) [inline README.md]: remove

Comment (rcaneill) [reply README.md]:
```suggestion
```

File: README.md (hunk 2 above)
Comment (rcaneill) [reply README.md]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (rcaneill) [reply README.md]:
```suggestion
```

File: README.md [hunk 4]
```diff
@@ -1,2 +1,93 @@
...
+- [ ] The need for the package is clear
+- [ ] All functions have documentation and associated examples for use
+- [ ] The package is easy to install
+
+
+#### Functionality
+
+- [ ] **Installation:** Installation succeeds as documented.
+- [ ] **Functionality:** Any functional claims of the software been confirmed.
+- [ ] **Performance:** Any performance claims of the software been confirmed.
```
Comment (rcaneill) [inline README.md]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -1,2 +1,93 @@
...
+- [ ] **Automated tests:**
+  - [ ] All tests pass on the reviewer's local machine for the package version submitted by the author. Ideally this should be a tagged version making it easy for reviewers to install.
+  - [ ] Tests cover essential functions of the package and a reasonable range of inputs and conditions.
```
Comment (rcaneill) [inline README.md]:
```suggestion
- [ ] **The package has tests:**
  - [ ] Tests cover essential functions of the package and a reasonable range of inputs and conditions.
```

File: README.md [hunk 6]
```diff
@@ -1,2 +1,93 @@
...
+- [ ] **Continuous Integration:** Has continuous integration setup (We suggest using Github actions but any CI platform is acceptable for review)
+- [ ] **Packaging guidelines**: The package conforms to the pyOpenSci [packaging guidelines](https://www.pyopensci.org/python-package-guide).
+    A few notable highlights to look at:
+    - [ ] Package supports modern versions of Python and not [End of life versions](https://endoflife.date/python).
+    - [ ] Code format is standard throughout package and follows PEP 8 guidelines (CI tests for linting pass)
```
Comment (rcaneill) [inline README.md]:
```suggestion
```

File: README.md [hunk 7]
```diff
@@ -1,2 +1,93 @@
...
+     
+
+### General checks
+
+- [ ] **Repository:** Is the source code for this software available at the [{{target-repository}}]({{target-repository}})?
```
Comment (rcaneill) [inline README.md]:
```suggestion
```

File: README.md [hunk 8]
```diff
@@ -1,2 +1,93 @@
...
+- [ ] **Contribution and authorship:** Has the submitting author ({{author-handle}}) made major contributions to the software? Does the full list of paper authors seem appropriate and complete?
+- [ ] **Substantial scholarly effort:** Does this submission meet the scope eligibility described in the [JOSS guidelines](https://joss.readthedocs.io/en/latest/submitting.html#substantial-scholarly-effort)
+- [ ] **Data sharing:** If the paper contains original data, data are accessible to the reviewers. If the paper contains no original data, please check this item.
+- [ ] **Reproducibility:** If the paper contains original results, results are entirely reproducible by reviewers. If the paper contains no original results, please check this item.
+- [ ] **Human and animal research:** If the paper contains original data research on humans subjects or animals, does it comply with [JOSS's human participants research policy and/or animal research policy](https://joss.readthedocs.io/en/latest/policies.html?highlight=animal#joss-policies)? If the paper contains no such data, please check this item.
```
Comment (rcaneill) [inline README.md]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -1,2 +1,93 @@
...
+
+
+### Functionality
+
+- [ ] **Installation:** Does installation proceed as outlined in the documentation?
+- [ ] **Functionality:** Have the functional claims of the software been confirmed?
+- [ ] **Performance:** If there are any performance claims of the software, have they been confirmed? (If there are no claims, please check off this item.)
```
Comment (rcaneill) [inline README.md]:
```suggestion
```

File: README.md [hunk 10]
```diff
@@ -1,2 +1,93 @@
...
+
+### Documentation
+
+- [ ] **A statement of need**: Do the authors clearly state what problems the software is designed to solve and who the target audience is?
```
Comment (rcaneill) [inline README.md]:
```suggestion
```

**Review by rcaneill — COMMENTED** [review]
I proposed changes to remove 1) the duplicates between JOSS and PyOpenSci and 2) too advanced checks

File: README.md (hunk 1 above)
Comment (vadmbertr) [reply README.md]: - I'm not sure if as a user I prefer to have examples over documentation
- Maybe "all user-facing" is too strict for us
- Personaly I would prefer documentation for the functions highlighted in the example(s)

File: README.md (hunk 2 above)
Comment (vadmbertr) [reply README.md]: yup!

File: README.md (hunk 3 above)
Comment (vadmbertr) [reply README.md]: yup

File: README.md [hunk 11]
```diff
@@ -1,2 +1,93 @@
...
+    - [ ] A [repostatus.org](https://www.repostatus.org/) badge,
+    - [ ] Python versions supported,
+    - [ ] Current package version (on PyPI / Conda).
+
+*NOTE: If the README has many more badges, you might want to consider using a table for badges: [see this example](https://github.com/ropensci/drake). Such a table should be more wide than high. (Note that the a badge for pyOpenSci peer-review will be provided upon acceptance.)*
+
+- [ ] Short description of package goals.
+- [ ] Package installation instructions
+- [ ] Any additional setup required to use the package (authentication tokens, etc.)
+- [ ] Descriptive links to all vignettes. If the package is small, there may only be a need for one vignette which could be placed in the README.md file.
```
Comment (vadmbertr) [inline README.md]: I would remove these as we remove similar checks before

File: README.md [hunk 12]
```diff
@@ -1,2 +1,93 @@
...
+    - [ ] Brief demonstration of package usage (as it makes sense - links to vignettes could also suffice here if package description is clear)
```
Comment (vadmbertr) [inline README.md]: I think this is important, and that's for the functions outlined there that I would like documentation (as I will likely use them as well)

File: README.md (hunk 4 above)
Comment (vadmbertr) [reply README.md]: yup

File: README.md (hunk 5 above)
Comment (vadmbertr) [reply README.md]: yup

File: README.md (hunk 6 above)
Comment (vadmbertr) [reply README.md]: agree as it's a bit too language specific for us

File: README.md (hunk 7 above)
Comment (vadmbertr) [reply README.md]: yep

File: README.md (hunk 8 above)
Comment (vadmbertr) [reply README.md]: yep

File: README.md (hunk 9 above)
Comment (vadmbertr) [reply README.md]: yep

File: README.md (hunk 10 above)
Comment (vadmbertr) [reply README.md]: yep

**Review by vadmbertr — COMMENTED** [review]
I just raised a point regarding documentation and examples.

I guess it's still too many of them and we have to prioritize now?

File: README.md (hunk 11 above)
Comment (rcaneill) [reply README.md]:
```suggestion
```

File: README.md (hunk 1 above)
Comment (rcaneill) [reply README.md]:
```suggestion
- [ ] **Vignette(s)** demonstrating major functionality that runs successfully locally.
- [ ] **Function Documentation:** for functions used in the examples.
```

Comment (rcaneill) [reply README.md]: @vadmbertr better?

**Comment by rcaneill** [comment]
> I guess it's still too many of them and we have to prioritize now?

Yes I agree. I removed the extra points so it may be easier to move forward now

**Comment by rcaneill** [comment]
The important main topics may be:
- general structure (having a readme, a licence, contributing guide, etc)
- tests
- doc

You see anything more?

**Comment by rcaneill** [comment]
@vadmbertr I updated and cleaned the checkboxes, what do you think?

**Comment by vadmbertr** [comment]
It's fine to me!

**Review by vadmbertr — APPROVED** [review]
Looks good!

**Pull request merged**